### ISSUE TITLE
HJSON file with only comments will error out

### ISSUE THREAD

**Issue by sonnyvan** [hjson/hjson-py#7]
This is a minor bug I've noticed. If the HJSON file contains only a comment, it will error out.
Using the HJSON linter at https://hjson.org/try.html, an empty object {} was created. 
```
Python 2.7.12 (default, Oct 11 2016, 05:20:59)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import hjson
>>> hjson.loads('//File with only comments')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sonnyvan/.virtualenvs/test/lib/python2.7/site-packages/hjson/__init__.py", line 173, in loads
    return _default_decoder.decode(s)
  File "/Users/sonnyvan/.virtualenvs/test/lib/python2.7/site-packages/hjson/decoder.py", line 514, in decode
    obj, end = self.raw_decode(s)
  File "/Users/sonnyvan/.virtualenvs/test/lib/python2.7/site-packages/hjson/decoder.py", line 558, in raw_decode
    raise e
hjson.scanner.HjsonDecodeError: Bad key name (eof): line 1 column 36 (char 35)
>>>
```

**Comment by sonnyvan** [comment]
Issue fixed in #8

**Comment by laktak** [comment]
I've pushed 2.0.7 to pypi
